application/config/packages.php

```diff
@@ -95,7 +95,8 @@
         ),
         'depends' => array(
             'limesurvey-public',
-            'embeddables'
+            'embeddables',
+            'bootstrap'
```

Comment:
Needed for this : https://github.com/LimeSurvey/LimeSurvey/pull/2839/files#diff-95fab772d475fc6bd9feb24f8a5e7a809747be3a57e932f0927e6dd7f5543fd9R277

Unsure if we need to move to theme.css ?